src/i18ndude/generator.py

```diff
@@ -10,6 +10,8 @@
 
 import re
 
+NOT_ALLOWED_IN_NAME = re.compile('\W')
```

Comment:
The report I received was about spaces, I generalized that. Did not find a authoritative reference about which characters *are* valid.

Reply:
See #71 comment, I verified the set of disallowed chars interactively in a reproduction scenario.